numpy/random/src/distributions/distributions.c

```diff
@@ -853,9 +854,30 @@ double random_vonmises(bitgen_t *bitgen_state, double mu, double kappa) {
        */
       s = (1. / kappa + kappa);
     } else {
-      double r = 1 + sqrt(1 + 4 * kappa * kappa);
-      double rho = (r - sqrt(2 * r)) / (2 * kappa);
-      s = (1 + rho * rho) / (2 * rho);
+#ifndef NP_RANDOM_LEGACY
+      /* Fallback to normal distribution for big values of kappa
+       * Fix only applies to Generator, not RandomState
+       * RandomState defines NP_RANDOM_LEGACY=1
+       */
+      if (kappa > 1e6) {
+        result = mu + sqrt(1. / kappa) * random_standard_normal(bitgen_state);
+        /* Check if result is within bounds */
+        if (result < -M_PI) {
+          return result + 2*M_PI;
+        }
+        if (result > M_PI) {
+          return result - 2*M_PI;
+        }
+        return result;
+      } else {
```

Comment:
Indentation needs to be 4 spaces, `else` on new line.

Reply:
Style needs fixing all round, let's leave that to a later cleanup.

Reply:
While I don't mind changing the commit, the entire file is formatted using these conventions (2 spaces, single like else).  IIRC think this is Google C format as formatted using clang-format.